### DISCUSSION TITLE
feat(codeEditor): add Completion Provider

### DISCUSSION THREAD

**Discussion by kimdiego2098** [dotnetcore/BootstrapBlazor#4474] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe the problem.

js-example: https://microsoft.github.io/monaco-editor/playground.html?source=v0.52.0#example-extending-language-services-completion-provider-example

![image](https://github.com/user-attachments/assets/4688867a-f034-48a9-a492-9868921deac6)

### Describe the solution you'd like

CodeEditor组件增加自动完成的功能

**Comment by bb-auto** [comment]
@kimdiego2098 Thank you for contacting us. We will give feedback later.
